### ISSUE TITLE
Adjust the height of the animation

### ISSUE THREAD

**Issue by zhiyao92** [Recouse/LinearProgressBar#12]
Hello, 

When the progress bar is adjust to the height of 2.0, the animation continue to remain as 1.0. Is there any option to increase the size of the animation as well?

**Comment by Recouse** [comment]
Hello,
Are you trying to adjust the value of `LinearProgressBar`'s `progressBarWidth` property?

**Comment by zhiyao92** [comment]
Hi Recouse, 

Thank you for your prompt reply. I am trying to adjust the height of the animation which is the CAShapeLayer. Currently if you resize the height of the LinearProgressBar to 2.0. The animation which is the CAShapeLayer continue to stay at 1.0.

**Comment by Recouse** [comment]
Fixed and pushed to master. You can try it now.

**Comment by zhiyao92** [comment]
Hi Recouse, 

I've pulled the latest changes, but I still have the same problem. I've uploaded an image to further illustrate this. Looking at the image, the lighter represent the background with the height of 2.0. And the darker red represent the animation which still stay at the height of 1.0. 

<img width="505" alt="Screen Shot 2019-11-25 at 4 33 23 PM" src="https://user-images.githubusercontent.com/14795793/69524586-56e1a080-0fa1-11ea-8c5f-bda1d4fcd3fb.png">
